corehq/apps/consumer_user/decorators.py

```diff
@@ -0,0 +1,16 @@
+from functools import wraps
+from django.contrib.auth.views import redirect_to_login
+from django.urls import reverse
+
+
+def login_required(view_func):
```

Comment:
Created a new decorator so that we could redirect to consumer_user login if the current user logged in user doesn't have corresponding ConsumerUser instance. ConsumerUserCaseRelationship has relation ship to ConsumerUser. Once he logins corresponding ConsumerUser will be created.